tests/integration/test_rest.py

```diff
@@ -36,3 +45,63 @@ async def test_get_event_success(joint_fixture: JointFixture, prepopped_events):
     )
     assert response.status_code == 200
     assert response.json() == expected
+
+
+async def test_msg_too_big_to_publish(
+    kafka: KafkaFixture, mongodb: MongoDbFixture, capsys
+):
+    """Test that a message that is too big to publish raises an error.
+
+    The error should be logged along with the request parameters.
+    """
+    kafka_size_limit = 10000
+    config = get_config(
+        sources=[kafka.config, mongodb.config], kafka_max_message_size=kafka_size_limit
+    )
+    configure_logging(config=config)
+
+    # Make test event that exceeds the Kafka size limit
+    test_event = StoredDLQEvent(
+        dlq_id=uuid4(),
+        headers={"correlation_id": str(uuid4())},
+        topic="test-topic",
+        type_="test-type",
+        key="test-key",
+        payload={"field": "x" * (kafka_size_limit * 2)},  # 2 times the max size
+        dlq_info=DLQInfo(
+            service="test-service",
+            partition=0,
+            offset=0,
+            exc_class="TestException",
+            exc_msg="Test message",
+        ),
+    )
+
+    # Insert test data
+    db_name = config.db_name
+    db = mongodb.client[db_name]
+    test_json = dto_to_document(test_event, id_field="dlq_id")
+    db["dlqEvents"].insert_one(test_json)
+
+    async with (
+        prepare_rest_app(config=config) as app,
+        AsyncTestClient(app=app) as client,
+    ):
+        # clear stderr/out buffers
+        out, err = capsys.readouterr()
```

Comment:
```suggestion
        _, _ = capsys.readouterr()
```
The values here aren't used, right? 
Could be explicit about it this way, but not strictly necessary